views/listings.php

```diff
@@ -0,0 +1,55 @@
+<?php
+/**
+ * Show available content from WP101.
+ *
+ * @global $api      An instance of WP101\API;
+ * @global $playlist An array of media from the WP101 API.
+ *
+ * @package WP101
+ */
+
+?>
+
+<div class="wrap" class="wp101-settings">
+	<h1>
+		<?php echo esc_html( _x( 'WordPress Video Tutorials', 'listings page title', 'wp101' ) ); ?>
```

Comment:
Could just reduce this to `esc_html_x` instead of sending it through multiple functions.